### PR TITLE
Kate/90812/To migrate turbos-info and payout-per-point to TS together with their tests + add tests for payout-per-point

### PR DESCRIPTION
**Pull request by kate-deriv**
## Changes:

- Migrate **turbos-info** and **payout-per-point** to TS (with tests).
- Add tests for **payout-per-point**.

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import React from 'react';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+
+type TTTurbosInfo = {
+    className?: string;
+};
+
+const TurbosInfo = observer(({ className }: TTTurbosInfo) => {
```
Comment (matin-deriv) [inline packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.tsx]:
```suggestion
const TurbosInfo = observer(({ className }: TTurbosInfo) => {
```

File: packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx [hunk 2]
```diff
@@ -23,7 +26,7 @@ const PayoutPerPoint = ({ currency, proposal_info, contract_type }) => {
                     icon='info'
                     is_bubble_hover_enabled
                     margin={0}
-                    zIndex={9999}
+                    zIndex={'9999'}
```
Comment (matin-deriv) [inline packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx]:
```suggestion
                    zIndex='9999'
```

Comment (maryia-deriv) [inline packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx]:
```suggestion
                    zIndex='9999'
```

File: packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 import { localize } from '@deriv/translations';
 import Fieldset from 'App/Components/Form/fieldset.jsx';
-import { connect } from 'Stores/connect';
+import { observer, useStore } from '@deriv/stores';
 
-const PayoutPerPoint = ({ currency, proposal_info, contract_type }) => {
+const PayoutPerPoint = observer(() => {
+    const {
+        modules: { trade },
+    } = useStore();
```
Comment (matin-deriv) [inline packages/trader/src/Modules/Trading/Components/Elements/Turbos/payout-per-point.tsx:11]: This is nice 👍🏼

**Review by maryia-deriv — APPROVED** [review]
@kate-deriv please check out Matin's comment

File: packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+
+type TTTurbosInfo = {
```
Comment (kate-deriv) [inline packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.tsx]:
```suggestion
type TTurbosInfo = {
```

**Review by matin-deriv — APPROVED** [review]

**Review by maryia-deriv — APPROVED** [review]
👍🏻

**Pull request merged**